src/sendReview.js

```diff
@@ -0,0 +1,27 @@
+export async function sendReview(author, rating, comment, movieID) {
```

Comment:
I would put this function on a CMS adapter object, which makes it easier to test.